src/cli/clear.test.js

```diff
@@ -0,0 +1,81 @@
+// @flow
+
+import fs from "fs";
+import path from "path";
+import tmp from "tmp";
+
+import {clear} from "./clear";
+import {run} from "./testUtil";
+import {help} from "./clear";
+
```

Comment:
Consider mocking out `rimraf`/`removePath` here—not for testing
convenience, but for safety in case the code under test or the test
harness is wrong. I could easily see a misconfigured test accidentally
removing the test runner’s actual SourceCred directory, which would be
quite bad.

Reply:
Updated.. trying to copy what @decentralion did w/ `exportGraph` in #1110.

Reply:
Yep, understood. The same general approach is good. It’s just that
`export-graph`’s primary effect is fairly benign (printing to stdout),
whereas a mistake in `clear` could actually be destructive to the host.